packages/@react-spectrum/link/docs/Link.mdx

```diff
@@ -64,6 +57,24 @@ import {Link as RouterLink} from 'react-router-dom';
 <Link><RouterLink to="/next-page">Next Page</RouterLink></Link>
 ```
 
+If the content is plain text, it will be styled and exposed to assistive technologies as a link.
+Events will need to be handled in JavaScript with the `onPress` prop.
+Note, this will not bring up link context menus in mobile, it's recommended that you use an anchor tag or something that will render one.
+
+```tsx example
+function TrackLink () {
```

Comment:
I think a simple alert is probably fine for this one.

Reply:
they're so disruptive :(